chains/evm/deployment/utils/types.go

```diff
@@ -0,0 +1,9 @@
+package utils
```

Comment:
Would like to avoid "types" files as they get bloated in scope.

With the suggested changes around inputting the full refs for MCM and timelock contract we shouldn't need this.